bundle.Dockerfile.konflux

```diff
@@ -0,0 +1,71 @@
+COPY LICENSE licenses/
+
+ARG CERT_MANAGER_OPERATOR_IMAGE=registry.redhat.io/cert-manager/cert-manager-operator-rhel9:v1.15.0 \
+    CERT_MANAGER_WEBHOOK_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9:v1.15.2 \
+    CERT_MANAGER_CA_INJECTOR_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9:v1.15.2 \
+    CERT_MANAGER_CONTROLLER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9:v1.15.2 \
+    CERT_MANAGER_ACMESOLVER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9:v1.15.2 \
+    KUBE_RBAC_PROXY_IMAGE=registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9:v4.17.0
+
+RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
+RUN ./render_templates.sh /manifests /metadata \
+    ${CERT_MANAGER_OPERATOR_IMAGE} \
+    ${CERT_MANAGER_WEBHOOK_IMAGE} \
+    ${CERT_MANAGER_CA_INJECTOR_IMAGE} \
+    ${CERT_MANAGER_CONTROLLER_IMAGE} \
+    ${CERT_MANAGER_ACMESOLVER_IMAGE} \
+    ${KUBE_RBAC_PROXY_IMAGE}
+
+FROM registry.redhat.io/rhel9-4-els/rhel-minimal:9.4
+
+ARG RELEASE_VERSION
+ARG COMMIT_SHA
+ARG SOURCE_URL
+
+# Core bundle labels.
+LABEL com.redhat.component="cert-manager-operator-bundle-container" \
+      name="cert-manager/cert-manager-operator-bundle" \
+      summary="Cert Manager support for OpenShift" \
+      description="Cert Manager support for OpenShift" \
+      distribution-scope="public" \
+      release="v${RELEASE_VERSION}" \
+      version="v${RELEASE_VERSION}" \
+      url="${SOURCE_URL}" \
+      maintainer="Red Hat, Inc." \
+      vendor="Red Hat, Inc." \
+      com.redhat.delivery.operator.bundle=true \
+      com.redhat.openshift.versions="v4.14-v4.18" \
+      io.openshift.expose-services="" \
+      io.openshift.build.commit.id="${COMMIT_SHA}" \
+      io.openshift.build.source-location="${SOURCE_URL}" \
+      io.openshift.build.commit.url="${SOURCE_URL}/commit/${COMMIT_SHA}" \
+      io.openshift.maintainer.product="OpenShift Container Platform" \
+      io.openshift.tags="openshift,cert,cert-manager,cert-manager-operator,tls" \
+      io.k8s.display-name="openshift-cert-manager-operator-bundle" \
+      io.k8s.description="cert-manager-operator-bundle-container" \
+      operators.operatorframework.io.bundle.mediatype.v1="registry+v1" \
+      operators.operatorframework.io.bundle.manifests.v1=manifests/ \
+      operators.operatorframework.io.bundle.metadata.v1=metadata/ \
+      operators.operatorframework.io.bundle.package.v1="openshift-cert-manager-operator" \
+      operators.operatorframework.io.bundle.channel.default.v1="stable-v1" \
+      operators.operatorframework.io.bundle.channels.v1="stable-v1,stable-v1.15" \
+      operators.operatorframework.io.metrics.builder="operator-sdk-v1.25.1" \
+      operators.operatorframework.io.metrics.mediatype.v1="metrics+v1" \
+      operators.operatorframework.io.metrics.project_layout="go.kubebuilder.io/v3"
+
```

Comment:
TODO (follow-up PRs would be fine for this)
- we need to make sure that the `make bundle` script updates these labels like it does for `bundle.Dockerfile` normally.

Reply:
Yeah, that will be different activity to update the labels added by `operator-sdk` tool. Because the tool doesn't add labels and licenses as required by the enterprise policy, and that was intention to have a different Dockerfile for konflux purpose.